src/components/calcite-panel/resources.ts

```diff
@@ -1,26 +1,40 @@
 export const CSS = {
+  backButton: "back-button",
   container: "container",
   header: "header",
-  headerLeadingContent: "header-leading-content",
+  heading: "heading",
+  summary: "summary",
   headerContent: "header-content",
-  headerTrailingContent: "header-trailing-content",
+  headerActionsEnd: "header-actions-end",
```

Comment:
I wonder if we want a class for `header-actions` and then modifiers `header-actions--end`, `header-actions--start`?

Reply:
Ah yeah...good call.

Reply:
Thanks @driskull. Updated.